themes/default/content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -973,23 +907,34 @@ Note that we must create this certificate in the `us-east-1` (Northern Virginia)
 ```typescript
 // Provision an SSL certificate to enable SSL -- ensuring to do so in us-east-1.
 const awsUsEast1 = new aws.Provider("usEast1", { region: "us-east-1" });
-const sslCert = new aws.acm.Certificate("sslCert", {
+const sslCert = new aws.acm.Certificate(
+  "sslCert",
+  {
```

Comment:
Is there a reason for this re-formatting?  The original looks more like standard TS formatting.

Reply:
No, probably an auto-formatter pluggin taking liberties.